### ISSUE TITLE
Snippets should be in fixed width font

### ISSUE THREAD

**Issue by pmorch** [raelgc/scudcloud#428]
In chome and firefox in the browser snippets are shown in fixed width font. They should in ScudCloud too, no?

What it looks like in chrome:
![chrome](https://cloud.githubusercontent.com/assets/159292/14490897/d437723c-0175-11e6-88d4-8d836acd5f7e.png)

What it looks like in scudcloud:
![scudcloud](https://cloud.githubusercontent.com/assets/159292/14490898/d820c45c-0175-11e6-87c3-66c91137e34b.png)

**Comment by raelgc** [comment]
Hi @pmorch and thanks for raising this issue.

In few minutes the update fixing this will be available. Sorry for the inconvenience!

**Comment by raelgc** [comment]
Just a friendly reminder that qtwebkit will use fonts available in the system, so while monospaced, probably they'll be a bit different than the browser one.

**Comment by pmorch** [comment]
Thanks for the swift fix!

For those like me that are using an older version (1.20-1) here on debian, I tried to apply 34614029791334f82c244fb93fa18b626e9f4e1b but the CSS selectors didn't work on that old version. Instead, this sledge hammer did it for me (the other fonts looked a little different, but that is ok with me):

``` diff
--- /usr/lib/python3/dist-packages/scudcloud/resources/resources.css.orig   2016-04-13 14:10:01.287931359 +0200
+++ /usr/lib/python3/dist-packages/scudcloud/resources/resources.css    2016-04-13 15:28:33.526605318 +0200
@@ -1,5 +1,7 @@
 nav.top, footer, #prefs_mac_ssb_tab {display: none !important;}
+/*
 * {font-family: Lato,"Helvetica Neue",Helvetica,"Segoe UI",Tahoma,Arial,sans-serif;}
+*/
 /* Fix to force display of filtered members while inviting for private channels (Fixes #280) */
 #channel_invite_container > .filter_select > .fsl_list_container > .monkey_scroll_wrapper {display: block !important;}
 /* Fix to force display of scrollbar in channel list (Fixes #309) */

```
